pubspec.yaml

```
@@ -1,20 +1,21 @@
 name: pact_dart
 description: Dart DSL for Pact, the de-facto API contract testing tool.
-version: 0.8.0
-homepage: https://github.com/matthewshirley/pact_dart
+version: 1.0.0
```

Let's keep the version the same. I can bump it when I publish the update.

```suggestion
version: 0.8.0
```